desktop/env.js

```diff
@@ -13,6 +13,8 @@ const dialog = require( 'electron' ).dialog;
  */
 const config = require( './lib/config' );
 const Settings = require( './lib/settings' );
+const EditorContextMenu = require( './lib/menu/editor-context-menu' );
+const GeneralContextMenu = require( './lib/menu/general-context-menu' );
```

Comment:
Minor: I think capitalization for modules is generally limited to modules with a constructor (or that use `new`).